communications/newsletters/newsletter_09_Jan2020.md

```diff
@@ -59,7 +61,9 @@ Do you have your _The Turing Way_ stories and experiences to share? Contact us t
 
 [Tweet](https://twitter.com/SoftwareSaved/status/1215572740381372416?s=20) from the official Twitter page of SSI.
 
-This year, among 18 [SSI fellowship]() recipients are our two highly contributing members, Sarah Gibson and Jez Cope. We are really thrilled to have their work and effort recognized by this prestigious fellowship. Sarah aims to [build a welcoming community of reproducible research](https://software.ac.uk/about/fellows/sarah-gibson) by encouraging contributions, promoting diversity and transparency, and fostering collaborations. Jez is interested in developing [infrastructure and practices to archive and preserve research data and software for the future](https://software.ac.uk/about/fellows/jez-cope). SSI fellowships are awarded every year to open science advocates for supporting their effort as ambassadors in their respective fields. More details in [this blog post](https://software.ac.uk/blog/2020-01-10-announcing-2020-software-sustainability-institute-fellows).
+This year, among 18 [SSI fellowship]() recipients are our two highly contributing members, Sarah Gibson and Jez Cope. We are really thrilled to have their work and effort recognized by this prestigious fellowship. 
+Sarah aims to [build a welcoming community of reproducible research](https://software.ac.uk/about/fellows/sarah-gibson) by encouraging contributions, promoting diversity and transparency, and fostering collaborations. 
+Jez is interested in developing [infrastructure and practices to archive and preserve research data and software for the future](https://software.ac.uk/about/fellows/jez-cope). SSI fellowships are awarded every year to open science advocates for supporting their effort as ambassadors in their respective fields. More details in [this blog post](https://software.ac.uk/blog/2020-01-10-announcing-2020-software-sustainability-institute-fellows).
 
```

Comment:
Should we also shout out to Tarek, Alex and William who attended book dashes like Kirstie did in her tweet? https://twitter.com/kirstie_j/status/1215672443785924610
I don't want to say that you need to attend a book dash to become a fellow, but it doesn't harm to highlight that book dash attendance is definitely beneficial 😄

Reply:
Ah, that's true. We definitely should! I like your argument <3

Reply:
Added!